derp/derp_server.go

```diff
@@ -157,15 +158,35 @@ type Server struct {
 	// local).
 	clientsMesh map[key.NodePublic]PacketForwarder
 	// sentTo tracks which peers have sent to which other peers,
-	// and at which connection number. This isn't on sclient
-	// because it includes intra-region forwarded packets as the
-	// src.
-	sentTo map[key.NodePublic]map[key.NodePublic]int64 // src => dst => dst's latest sclient.connNum
+	// at which connection number, and how many packets/bytes were
+	// sent per connection. This isn't on sclient because it
+	// includes intra-region forwarded packets as the src.
+	sentTo map[key.NodePublic]map[key.NodePublic]flowData
 
 	// maps from netip.AddrPort to a client's public key
 	keyOfAddr map[netip.AddrPort]key.NodePublic
 }
 
+// Per connection statistics for observability of who is using the
+// server and why, including intra-region forwarded packets, kept in
+// the sentTo map in the server. Accessed only under s.mu.
+type flowData struct {
+	// TODO: why is connNum signed?
+	connNum int64  // unique number per src->dest connection
+	pkts    uint64 // total packets sent on this connection
+	bytes   uint64 // total bytes
+}
```

Comment:
you'll also want to track a last seen time, so you can clean up old flows not seen in N minutes